package.json

```diff
@@ -118,11 +118,11 @@
   "prettier": "@nordicsemiconductor/asset-tracker-cloud-code-style/.prettierrc",
   "dependencies": {
     "@aws-lambda-powertools/metrics": "1.12.1",
-    "@hello.nrfcloud.com/proto": "5.5.4",
+    "@hello.nrfcloud.com/proto": "5.5.10",
     "@middy/core": "4.6.1",
     "@nordicsemiconductor/from-env": "2.0.0",
     "@nordicsemiconductor/timestream-helpers": "5.0.0",
-    "@sinclair/typebox": "0.31.5",
+    "@sinclair/typebox": "0.31.14",
```

Comment:
Unrelated change, revert.

Reply:
This change is required to satisfy tsc because all typebox modules must be the same version.